src/system/Form/Radio/Radio.tsx

```diff
@@ -0,0 +1,97 @@
+/** @jsxImportSource theme-ui */
+
+import classNames from 'classnames';
+import React, { Ref, forwardRef, useCallback } from 'react';
+import { ThemeUIStyleObject } from 'theme-ui';
+
+import { RadioOption } from './RadioOption';
+
+export const VIP_RADIO = 'vip-radio-component';
+
+export type RadioOptionOptionProps = {
+	id: string;
+	value: string;
+	disabled?: boolean;
+	className?: string;
+	label?: string;
+	renderLabel?: ( props, labelStyles: ThemeUIStyleObject ) => JSX.Element | null;
```

Comment:
why `| null`? if you left it undefined you could give it a default value of `() => null` and remove the ternaries